spec/models/user_spec.rb

```diff
@@ -2905,5 +2905,18 @@ def reset_last_seen_cache!(user)
       expect(moderator.last_seen_reviewable_id).to eq(mod_reviewable.id)
       expect(user.last_seen_reviewable_id).to eq(group_reviewable.id)
     end
+
+    it "publishes a message to the user's /reviewable_counts message bus channel" do
+      user.update!(admin: true)
+      reviewable = Fabricate(:reviewable)
```

Comment:
We don’t use this variable, so we can probably just call `Fabricate(:reviewable)`. It would be best to have all the setup in a `before` block but I’m not sure there’s a quick way to do that without revamping the whole spec for this method :sweat_smile:

Reply:
oops my bad!

Reply:
No worries :slightly_smiling_face: :+1: